.circleci/config.yml

```diff
@@ -21,9 +21,10 @@ jobs:
           command: ./build_and_push.sh
           environment:
             TAG_SUFFIX: ""
-            CAIRO_VERSION: 0.11.2
+            CAIRO_VERSION: "v0.11.2"
             OZ_VERSION: 0.5.1
             CAIRO_COMPILER_TARGET_TAG: "v1.1.0"
+            ASSET_NAME: "release-x86_64-unknown-linux-musl.tar.gz"
```

Comment:
Rename `ASSET_NAME` to `CAIRO_COMPILER_ASSET_NAME`

Reply:
https://github.com/0xSpaceShard/cairo-cli-docker/pull/19/commits/f4e8f9b0916cbad645b2bc8ea5df34a4a2c7ec5b